program/rust/src/processor.rs

```diff
@@ -71,7 +72,18 @@ pub fn process_instruction(
         UpdTest => Err(OracleError::UnrecognizedInstruction.into()),
         SetMinPub => set_min_pub(program_id, accounts, instruction_data),
         UpdPriceNoFailOnError => upd_price_no_fail_on_error(program_id, accounts, instruction_data),
-        ResizePriceAccount => resize_price_account(program_id, accounts, instruction_data),
+        ResizePriceAccount => {
+            #[cfg(feature = "price_v2_resize")]
```

Comment:
Yes, we should delete this code

Reply:
Yes, after talking to Guillermo I noticed that I overlooked a simple solution - i.e. we can prevent adding v1 prices altogether under the `pythnet` feature since we're changing the oracle anyway - meaning no straggler v1 accounts would ever be created.